lib/collection/src/collection/mod.rs

```diff
@@ -531,16 +531,33 @@ impl Collection {
                     sync: true,
                     method: None,
                 };
+
                 if check_transfer_conflicts_strict(&transfer, transfers.iter()).is_some() {
                     continue; // this transfer won't work
                 }
+
+                // TODO: Should we, maybe, throttle/backoff this requests a bit?
```

Comment:
In current implementation it doesn't create any additional requests, in case of the offline peer. It just replaces request to consensus with this `health_check`. So I think it is ok to keep this part as-is in this PR